frontend/public/components/_terminal.scss

```diff
@@ -11,4 +11,14 @@
   height: 100%;
   padding: 0;
   background-color: $color-pf-black;
+  position: relative;
+  z-index: 1040; // to get above 'terminal xterm fullscreen's z-index of 255 and $zindex-navbar-fixed of 1030
```

Comment:
worth commenting this is specifically to cover these elements in fullscreen mode

Reply:
fixed